frontend/src/routes/ClusterManagement/Clusters/components/ClusterPolicySidebar.tsx

```diff
@@ -49,6 +49,72 @@ const useStyles = makeStyles({
             textDecoration: 'underline',
         },
     },
+    backAction: {
+        fontSize: '14px',
+        color: 'var(--pf-global--Color--400)',
+        paddingBottom: '16px',
+        '& button': {
+            border: 0,
+            cursor: 'pointer',
+            background: 'none',
+            color: 'var(--pf-global--link--Color)',
+        },
+    },
+    detailTitleGroup: {
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+    titleText: {
+        fontSize: '20px',
+        color: 'var(--pf-global--Color--100)',
+    },
+    detailActions: {
+        '& button': {
+            backgroundColor: 'transparent',
+            fontSize: '16px',
+            '&::before': {
+                border: '0',
+            },
+            '& span': {
+                color: 'var(--pf-global--Color--100)',
+            },
+        },
+    },
+    detailText: {
+        fontSize: '16px',
+        lineHeight: '21px',
+        color: 'var(--pf-global--Color--200)',
+        padding: '22px 0',
+    },
+    // knowledgebaseLink: {
+    //     border: 0,
+    //     cursor: 'pointer',
+    //     background: 'none',
+    //     color: 'var(--pf-global--link--Color)',
+    //     padding: 0,
+    // },
+    subDetailContainer: {
+        display: 'grid',
+        gridTemplateColumns: '1fr 1fr 1fr',
+        columnGap: '10px',
+        padding: '27px 0 20px 0',
+    },
+    subDetail: {
+        color: 'var(--pf-global--Color--100)',
+        '& > div': {
+            display: 'flex',
+            paddingBottom: '16px',
+            '& h3': {
+                fontSize: '14px',
+                fontWeight: 700,
+                lineHeight: '18px',
+            },
+            '& svg': {
+                alignSelf: 'center',
+                marginRight: '10px',
+            },
+        },
+    },
```

Comment:
Why is there so much custom CSS needed for this design?

Reply:
Planning to cut this down with future design iterations when we are no longer using mock data.